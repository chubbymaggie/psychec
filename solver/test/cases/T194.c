void f() {
    int a;
}
void g() {
    a = "foo";
}
