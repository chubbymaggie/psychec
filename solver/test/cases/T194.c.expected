/* Forward declarations */

/* Type definitions */

/* Variables and functions */
 char* a ; 

